features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx

```diff
@@ -472,8 +472,7 @@ class InfoBar extends React.Component {
      */
     componentDidMount() {
         const api = new Api();
-        const promised_api = api.getAPIById(this.api_uuid);
-        promised_api
+        api.getAPIById(this.api_uuid)
```

Comment:
I think we could use the API UUID from the props directly here in `componentDidMount`, No need to assign it to `this`

Reply:
Fixed!